<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Kirollos/Downloads/icons8-git-48.png</normaloff>C:/Users/Kirollos/Downloads/icons8-git-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Welcome</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Welcome to my Git App&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>C:/Users/Kirollos/Downloads/git-seeklogo.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Enter your Git username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_username"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Enter your Git email:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_email"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pb_go_function">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Go to Git Functions</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Git Functions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Please enter the project directory :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="LE_Directory"/>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Browse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pb_git_init">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git init</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="12">
         <widget class="QTextEdit" name="Terminal"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pb_git_commit">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git commit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pb_git_push">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git push</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pb_git_pull">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git pull</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pb_git_clone">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git clone</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pb_git_reset">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git reset</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pb_git_status">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git status</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="pb_git_log">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git log</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pb_git_branch">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git branch</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="pb_create_branch">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>create branch</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="pb_gotobranch">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>go to branch</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="pb_gitmerge">
          <property name="styleSheet">
           <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);</string>
          </property>
          <property name="text">
           <string>git merge</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="pb_Back">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QPushButton" name="pb_Clear">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(62, 44, 0);
</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
